presto-docs/src/main/sphinx/release/release-0.225.rst

```diff
@@ -0,0 +1,50 @@
+=============
+Release 0.225
+=============
+
+General Changes
+_______________
+* Fix under-reporting of output data size from a fragment. (:issue:`11770`)
+* Fix a bug where spatial joins would cause workers to run out of memory and crash. (:pr:`13251`)
+* Fix leak in operator peak memory computations. (:issue:`13210`)
+* Allow overriding session zone sent by clients. (:issue:`13140`)
+* Throw an exception when a spatial partition is made with 0 rows.
+* Add ``TableStatistics`` field containing statistics estimates for input tables to the ``QueryInputMetadata`` in ``QueryCompletedEvent`` for join queries. (:pr:`12808`)
+* Add configuration property ``experimental.max-total-running-task-count`` and ``experimental.max-query-running-task-count``
+  to limit number of running tasks for all queries and a single query, respectively. Query will be killed only if both conditions are violated. (:pr:`13228`)
+
+JDBC Changes
+____________
+* Match schema and table names case insensitively. This behavior can be enabled by setting the ``case-insensitive-name-matching`` catalog configuration option to true.
+
+Web UI Changes
+______________
+* Display tasks per stage on the query page.
+
+Elasticsearch Connector Changes
+_______________________________
+* Fix and support nested fields in Elasticsearch Connector. (:issue:`12642`)
+
+Hive Changes
+____________
+* Improve performance for positioned reads of files on S3. (:pr:`13222`)
```

Comment:
@aweisberg would a user know whether they are doing a positioned read or not? If not, I think we should just say

`Improve performance of file reads on S3. (:pr:`13222`)`

Reply:
Updated after confirm with Ariel